python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/community_lists.schema.yml

```diff
@@ -8,8 +8,14 @@ type: dict
 keys:
   community_lists:
     type: list
+    description: If both `community_lists` and `ip_community_lists` are defined, then `ip_community_lists` takes precedence.
     primary_key: name
     display_name: Community Lists (legacy model)
+    deprecation:
+      warning: true
+      removed: false
```

Comment:
```suggestion
```

Reply:
@Shivani-gslab we need to keep `removed: false` here as we cannot remove the key without deprecating it first in a minor release to follow SemVer (hence we deprecate now and will remove in 6.0.0)

Reply:
Got it, but the default itself is false, so we can skip this in schema.

Reply:
Done.